public/index.html

```diff
@@ -42,6 +67,11 @@
         console.log('handshake');
       });
 
+      socket.on('sv_refresh_rooms', function(rooms) {
+        console.log('roomList');
+        $('#room-list').html(roomListTmpl.render({roomList: rooms}));
+      });
+
```

Comment:
Server does not emit `sv_refresh_rooms` event.
We should be able to safely remove this.